.tekton/tasks/e2e-test.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       - name: APP_SUFFIX
         value: "$(params.app_suffix)"
       - name: COMPONENT_REPO_URLS
-        value: "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic,https://github.com/redhat-appstudio-qe/retrodep,https://github.com/cachito-testing/pip-e2e-test,https://github.com/redhat-appstudio-qe/fbc-sample-repo,https://github.com/redhat-appstudio-qe/nodejs-no-dockerfile,https://github.com/redhat-appstudio-qe/maven-hello-world,https://github.com/redhat-appstudio-qe/source-build-parent-image-with-digest-only,https://github.com/redhat-appstudio-qe/source-build-parent-image-with-both-tag-digest,https://github.com/redhat-appstudio-qe/source-build-use-latest-parent-image,https://github.com/redhat-appstudio-qe/source-build-parent-image-from-registry-rh-io"
+        value: "https://github.com/redhat-appstudio-qe/devfile-sample-python-basic,https://github.com/redhat-appstudio-qe/retrodep,https://github.com/cachito-testing/pip-e2e-test,https://github.com/redhat-appstudio-qe/fbc-sample-repo,https://github.com/redhat-appstudio-qe/nodejs-no-dockerfile,https://github.com/redhat-appstudio-qe/maven-hello-world,https://github.com/redhat-appstudio-qe/source-build-parent-image-with-digest-only,https://github.com/redhat-appstudio-qe/source-build-use-latest-parent-image,https://github.com/redhat-appstudio-qe/source-build-parent-image-from-registry-rh-io,https://github.com/redhat-appstudio-qe/source-build-base-on-konflux-image"
```

Comment:
Would like to understand, how [this image](https://github.com/redhat-appstudio-qe/source-build-base-on-konflux-image/blob/df38dbba893267deacbf278b527e932c2cc05902/Dockerfile#L1) used in Dockerfile is getting update?

Reply:
Generally, that image will not be updated. Unless it is really necessary to rebuild the image for any future tests.

Reply:
Got it.. one more question will keep it here.. why are we removing [this repo](https://github.com/redhat-appstudio-qe/source-build-parent-image-with-both-tag-digest) from the test? is this PR combined with some other changes?

Reply:
Because the parent image in the new test repo is also in the form having both tag and digest.